tox.ini

```diff
@@ -69,7 +69,7 @@ usedevelop = false
 [testenv:format]
 commands =
     yapf -i -r molecule/ test/
-deps = yapf>=0.25.0,<0.27
+deps = yapf>=0.25.0,<0.27  # pyup: < 0.27 # disable updates, conflicts with flake8 as per https://github.com/ansible/molecule/pull/1915
```

Comment:
oh.. this one seems wrong because it's parsed as a single line. we should turn it into a list